### PR TITLE
Fix Cosmos session key login

### PR DESCRIPTION
**Pull request by raykyri**
- ESM imports don't work on server, use stringify directly instead of importing from canvas actions/sessions.
    - Fixes Cosmos session key verification on heroku envs
- Add back Cosmos API changes that were removed because of tests
    - Fixes Cosmos magic link logins

## Link to Issue
Closes: #TODO

## Description of Changes
- Adds FIXME widget to TODO page.
- 

## Test Plan
- Unit tested the `FIXME()` call.
- CA (click around) tested on local and frack:
  - TODO page
  - 

## Deployment Plan

1. 

## Other Considerations

-

### CODE REVIEW

File: packages/commonwealth/server/util/cosmosProxy.ts [hunk 1]
```diff
@@ -103,6 +104,133 @@ function setupCosmosProxy(app: Express, models: DB) {
...
+        if (!targetUrl) {
+          const fallback = await models.Chain.findOne({
+            where: { id: 'osmosis' },
+            include: models.ChainNode,
+          });
+          targetUrl = fallback.ChainNode.alt_wallet_url;
+        }
+        log.trace(`Found cosmos endpoint: ${targetUrl}`);
+
+        const response = await axios.get(targetUrl + '/node_info', {
```
Comment (jnaviask) [inline packages/commonwealth/server/util/cosmosProxy.ts:140]: Why always node_info?

**Comment by raykyri** [comment]
This just seems to be what Magic expects; in the code where they post they look for a specific JSON field that shows up on node_info.On Sep 19, 2023, at 11:46 AM, Jake Naviasky ***@***.***> wrote:﻿
@jnaviask commented on this pull request.



In packages/commonwealth/server/util/cosmosProxy.ts:
> +          include: models.ChainNode,
+        });
+        if (!chain) {
+          throw new AppError('Invalid chain');
+        }
+        let targetUrl = chain.ChainNode?.alt_wallet_url;
+        if (!targetUrl) {
+          const fallback = await models.Chain.findOne({
+            where: { id: 'osmosis' },
+            include: models.ChainNode,
+          });
+          targetUrl = fallback.ChainNode.alt_wallet_url;
+        }
+        log.trace(`Found cosmos endpoint: ${targetUrl}`);
+
+        const response = await axios.get(targetUrl + '/node_info', {

Why always node_info?

**Comment by jnaviask** [comment]
@raykyri then why does the body of the request to the RPC endpoint make a "status" method call?

**Comment by jnaviask** [comment]
Superseded by #5143.

**Pull request closed without merging**